lib/core/src/control/io/mouse.rs

```diff
@@ -1,407 +1,144 @@
-//! This module contains the `MouseManager` implementation, its associated structs such as
-//! `MousePositionEvent`, `MouseClickEvent` and `MouseWheelEvent`.
+//! This module contains implementation of a oldmouse manager and related utilities.
```

Comment:
Is this correct?

Reply:
I will fix the `oldmouse` word introduced in the comments.